jaqpotpy/models/trainers/base_trainer.py

```diff
@@ -6,7 +6,8 @@
 import requests
```

Comment:
instead of commenting out and keeping old files in the repository you can delete them because:
1. We probably won't need any of those
2. If we need them we have the old files on github and we can bring them back anytime